### DISCUSSION TITLE
Slider to stream bug

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#16990] [Q&A]
when i use slider to stream tool it adds an extra note at where the slider begins, but as the last note in the stream; only happens on some slider velocities (5 & 10, maybe more i didn't check) and changing slider size and timeline position doesn't seem to help.

video of it happening (too large for github)
https://youtu.be/Mau3GGDwnfU

[logs.zip](https://github.com/ppy/osu/files/8144736/logs.zip)

**Comment by Axilotl17** [comment]
oh and; i have only noticed this in 1/6 time, maybe it happens in 1/4 too im unsure

**Comment by bdach** [comment]
Could you please link the map & timestamp for the slider that this is happening on?

**Reply by Axilotl17** [reply]
Its a map im making, and its on all maps, so i dont think that would help much. i did manage to recreate it, and its only on some spots, for some reason; here's what i found

create any slider on 1/6 time, at velocity 5; this seems to work on all maps. i noticed some times it just doesn't happen, sometimes it does. im unsure, although i can manage to recreate it 80% of the time, but it does work on all maps.

**Reply by bdach** [reply]
I've tried following your instructions on several sliders on at least 3 maps and cannot get this bug to reproduce. Please link at least one map where it's present.

As an aside, I do note that you're using linux - do you happen to be using pipewire by any chance? Asking because this may be related to some funky behaviour on some linuxen wherein something will screw with floating point precision which may lead to issues like this.
